server/model/agent.go

```diff
@@ -28,13 +35,59 @@ type Agent struct {
 	Capacity    int32  `json:"capacity"      xorm:"capacity"`
 	Version     string `json:"version"       xorm:"'version'"`
 	NoSchedule  bool   `json:"no_schedule"   xorm:"no_schedule"`
+	// OrgID is counted as unset if set to -1, this is done to ensure a new(Agent) still enforce the OrgID check by default
+	OrgID int64 `json:"org_id"        xorm:"INDEX 'org_id'"`
+	// RepoID is counted as unset if set to -1, this is done to ensure a new(Agent) still enforce the OrgID check by default
+	RepoID int64 `json:"repo_id"       xorm:"INDEX 'repo_id'"`
+	// Server side enforced agent filters
+	Filters map[string]string `json:"filters" xorm:"'filters' json"`
 } //	@name Agent
 
+const (
+	SystemAgentOwnerID = -1
```

Comment:
As it is used to signal repo / org is not set as well sth like `IDNotSet` would be better:

```suggestion
	IDNotSet = -1
```